scripts/generate_secrets.py

```diff
@@ -2,6 +2,52 @@
 import yaml
```

Comment:
Worth sticking a #!/usr/bin/env python line at the top so the script can be executed from a shell without having to prefix it with "python"?